external-builds/pytorch/patches/pytorch/v2.7.0/pytorch/hipified/0001-Rework-LoadHIP.cmake-to-be-based-purely-on-CMAKE_PRE.patch

```diff
@@ -1,7 +1,7 @@
-From a614a3c8bec3e8392f88b02498eaec9255baeaf0 Mon Sep 17 00:00:00 2001
+From 502e9e63887274f3fa5f975a7990addea61cdb93 Mon Sep 17 00:00:00 2001
 From: Stella Laurenzo <stellaraccident@gmail.com>
 Date: Mon, 17 Feb 2025 16:47:57 -0800
-Subject: [PATCH 01/13] Rework LoadHIP.cmake to be based purely on
+Subject: [PATCH 01/12] Rework LoadHIP.cmake to be based purely on
  CMAKE_PREFIX_PATH.
 
```

Comment:
Note-to-reviewers: this patch is not actually changed.

Reply:
If people would prefer, I could send PRs modifying patches differently.

Option A: Only push modified patch files, let old patches be inaccurate
Option B: Send one commit just modifying patch files, send a second commit that rebasing/updates all patch files. Merge with 'squash and merge' as usual